swagger2.0/oic.r.cloudopenapi.swagger.json

```diff
@@ -722,49 +722,50 @@
       }
     },
     "DeviceWithLinks": {
-      "allOf": [
-        {
-          "$ref": "#/definitions/Device"
+      "type": "object",
+      "properties": {
+        "device": {
+          "$ref": "https://raw.githubusercontent.com/ondrejtomcik/coreocf/Bugzilla-2709/swagger2.0/oic.wk.d.swagger.json#/definitions/Device"
         },
-        {
+        "devicestatus": {
           "type": "object",
           "properties": {
-            "links": {
-              "type": "array",
-              "items": {
-                "$ref": "https://raw.githubusercontent.com/ondrejtomcik/coreocf/Bugzilla-2709/swagger2.0/oic.wk.res.swagger.json#/definitions/oic.oic-link"
-              }
+            "s": {
+              "$ref": "#/definitions/DeviceStatus"
             }
+          }
+        },
+        "links": {
+          "type": "array",
+          "items": {
+            "$ref": "https://raw.githubusercontent.com/ondrejtomcik/coreocf/Bugzilla-2709/swagger2.0/oic.wk.res.swagger.json#/definitions/oic.oic-link"
+          }
+        }
+      },
+      "example": {
+        "device": {
+          "dmn": "Open Connectivity Foundation",
+          "n": "Food safety sensor",
+          "di": "53080a4f-5e3e-4291-802f-3436238232d2"
+        },
+        "devicestatus": {
+          "s": "online"
+        },
+        "links": [
+          {
+            "href": "/53080a4f-5e3e-4291-802f-3436238232d2/humidity",
```

Comment:
As discussed, remove the uuid.

Reply:
Why in this case? These are the links from the RD, not from the end Device.